WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailActivity.java

```diff
@@ -176,6 +166,7 @@ public void showStatsActivityForSite(int localTableSiteId, NoteBlockRangeType ra
             Intent intent = new Intent(this, StatsViewAllActivity.class);
```

Comment:
I didn't do this because of all of the arguments the activity needs, and I doubt it'd be used elsewhere to show a list of followers.